docs/getting-started-bankid.md

```diff
@@ -27,15 +27,14 @@ These are only necessary if you plan to store your certificates in Azure KeyVaul
 
 [![Deploy to Azure](https://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FActiveLogin%2FActiveLogin.Authentication%2Fmaster%2Fsamples%2FAzureProvisioningSample%2FActiveLogin.json%23)
 
-1. Deploy Azure KeyVault to yur subscription, the ARM-template available in [AzureProvisioningSample](https://github.com/ActiveLogin/ActiveLogin.Authentication/tree/master/samples/AzureProvisioningSample) contains a KeyVault, so preferably you deploy that.
+1. Deploy Azure KeyVault to your subscription. The ARM-template available in [AzureProvisioningSample](https://github.com/ActiveLogin/ActiveLogin.Authentication/tree/master/samples/AzureProvisioningSample)  contains configuration that creates a KeyVault and enables [Managed Service Identity](https://azure.microsoft.com/en-us/resources/samples/app-service-msi-keyvault-dotnet/) for the App Service.
 1. [Import the certificates](https://docs.microsoft.com/en-us/azure/key-vault/certificate-scenarios#import-a-certificate) to your Azure Key Vault.
-1. Add your KeyVault setup to your config. The AD client should have access to the KeyVault certificate specified in `AzureKeyVaultSecretIdentifier`.
+1. Add the following to your config, the secret identifier is expected in the form of `"https://keyvaultname.vault.azure.net/secrets/keyidentifier"` .
```

Comment:
Please make it more clear what parts are "variables", like this:
`"https://[keyvaultname].vault.azure.net/secrets/[keyidentifier]"`